lib/foreman_bootdisk/engine.rb

```diff
@@ -31,12 +31,10 @@ class Engine < ::Rails::Engine
         requires_foreman '>= 1.9'
 
         security_block :bootdisk do |map|
-          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :help],
+          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :subnet, :help],
                                           :'foreman_bootdisk/api/v2/disks' => [:generic, :host]}
         end
 
-        role "Boot disk access", [:download_bootdisk]
```

Comment:
This was my attempt to provide saner default role for new users. Bootdisk role by itself is unusable as you cannot generate host-images and subnet-images without this permission (either unfiltered or with a filter).

Seed script was to workaround chicken and egg problem, but I found a different solution in https://github.com/theforeman/foreman/pull/2572.

Anyway I am removing this from this patch and leaving this as is.